Missing regexTypes.d.ts in distributed module

There seems to be a problem with the distribution of the [`regexTypes.d.ts`](https://github.com/sql-formatter-org/sql-formatter/blob/master/src/lexer/regexTypes.d.ts) file in the module available on npm.

When typescripts checks a project using the library, it complains:

```
node_modules/sql-formatter/lib/src/lexer/Tokenizer.d.ts:2:29 - error TS2307: Cannot find module './regexTypes' or its corresponding type declarations.

2 import * as regexTypes from "./regexTypes";
                              ~~~~~~~~~~~~~~


Found 1 error in node_modules/sql-formatter/lib/src/lexer/Tokenizer.d.ts:2
```

`lib/src/lexer` indeed does not contain the requisite file. It appears that the compilation step considers it as a plain typescript file and emits an (empty) module in `lib/lexer` instead:

```console
$ find node_modules/sql-formatter -name 'regexTypes.*'
node_modules/sql-formatter/lib/lexer/regexTypes.d.js
node_modules/sql-formatter/lib/lexer/regexTypes.d.js.map
```

The issue appears in (at least) 8.2 and 8.1.

--- a/src/lexer/Tokenizer.ts
+++ b/src/lexer/Tokenizer.ts
@@ -1,6 +1,6 @@
 import { Token, TokenType } from 'src/lexer/token';
 import * as regex from 'src/lexer/regexFactory';
-import * as regexTypes from 'src/lexer/regexTypes';
+import type * as regexTypes from 'src/lexer/regexTypes';
 
 import { escapeRegExp } from './regexUtil';
 import TokenizerEngine, { type TokenRule } from './TokenizerEngine';

